terraform/shared/modules/env/clamav.tf

```diff
@@ -11,11 +11,9 @@ module "clamav" {
 
   cf_org_name   = var.cf_org_name
   cf_space_name = var.cf_space_name
-  clamav_image  = "ajilaag/clamav-rest:20230228"
+  clamav_image  = "ghcr.io/gsa-tts/fac/clamav:latest"
```

Comment:
(Non-Blocking)

We can use YYYYMMDD if we want here, however, the assumption is that `:latest` will always be a "clean" image, as in order for it to get into our GHCR, it needs to have 0 scanned items via trivy.

Reply:
For the purposes of this PR, I am just replicating what was in the workflow.

I figured we can pin the ref to a specific date by making a PR on this line if we ever have a problem unrelated to scanning.